basicdataset/src/main/java/ai/djl/basicdataset/cv/CocoDetection.java

```diff
@@ -79,6 +80,12 @@ public PairList<Long, Rectangle> getObjects(long index) {
         return labels.get(Math.toIntExact(index));
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public List<String> getClasses() {
+        return synset;
```

Comment:
It is better to fail clearly rather than have something partial that may confuse people that it works. Here, just throw an exception, specifically at UnsupportedOperationException to indicate that it is not yet implemented.